bin/compile

```diff
@@ -115,16 +113,12 @@ expandVer() {
     echo $(<${DataJSON} jq -r 'if .Go.VersionExpansion."'${v}'" then .Go.VersionExpansion."'${v}'" else "'${v}'" end')
 }
 
-# Report deprecated versions to user
+# Report development versions to user
 # Use after expandVer
 reportVer() {
     local ver="${1}"
     mcount "version.go.$ver"
 
-    if <"${DataJSON}" jq  -e '.Go.Supported | any(. == "'${ver}'")' &> /dev/null; then
```

Comment:
Maybe we change this to supported major versions instead? I think it's worth telling people who are still running anything older than go1.11 that they are building using an unsupported version of Go.

Reply:
I will add this as a separate feature.